src/main/java/com/minecolonies/coremod/colony/buildings/AbstractBuildingStructureBuilder.java

```diff
@@ -669,7 +674,7 @@ public void checkOrRequestBucket(@Nullable final BuilderBucket requiredResources
                         break;
                     }
 
-                    worker.createRequestAsync(new Stack(itemStack.getItemStack(), requestCount, requestCount));
+                    worker.createRequestAsync(new Stack(itemStack.getItemStack(), Math.max(requestCount, resourceBatchSize), requestCount));
```

Comment:
That looks simpler than what I would've expected even.
I would make the resourceBatchSize a getter which on default returns 1  (in the abstract building of the structure builders) and in the miner returns 10.

Such that here we request requestCount * getResourceBatchSize

Reply:
I like it.

Reply:
This way instead of requesting 32 planks and 32 fences, we request 30/40 (which is a better ratio considering what we need too)